src/components/ebay-lightbox-dialog/index.marko

```diff
@@ -8,9 +8,17 @@ $ var buttonPosition = input.buttonPosition || 'right';
     on-open("emit", "open")
     on-close("emit", "close")
     button-position=buttonPosition
-    class=[input.class, "lightbox-dialog--mask-fade"]
+    class=[input.class, "lightbox-dialog--mask-fade", input.expressiveSrc && "lightbox-dialog--expressive"]
```

Comment:
Looks great!!! The only thing is that I think "expressive" isn't very descriptive for an arg. Since this is more or less a banner image, it's probably preferable to use that in the name to make it more intuitive to use. 

Maybe change the args to something like, `bannerImgSource` and `bannerImgPosition`?

Reply:
Good idea, thanks Arthur. I updated the names